src/codeu/chat/common/SQLFormatter.java

```diff
@@ -41,34 +40,33 @@ public static String sqlID(Uuid userID1, Uuid userID2) {
       sqlID2 = sqlID2.replace("]", "");
     }
 
-    String sqlID = "'" + sqlID1 + sqlID2 + "'";
+    String sqlID = sqlID1 + sqlID2;
     return sqlID;
   }
 
   public static String sqlName(String userName) {
-    String sqlName = "'" + userName + "'";
+    String sqlName = userName;
```

Comment:
Just remove these string methods that don't do anything but return the parameter.